### PR TITLE
Make the LicensesDialogFragment's licenses text styleable

### PR DESCRIPTION
**Pull request by ChaosLeong**
Because the default css style is too bright in __night mode__, so I make the __LicensesDialogFragment__'s licenses text can be styled.

* Update dependencies
* Make LicensesDialogFragment's license text styleable
* Add samples for coustom css style

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -70,25 +70,32 @@
...
-        <maven-javadoc-plugin.version>2.10.3</maven-javadoc-plugin.version>
-        <maven-source-plugin.version>3.0.0</maven-source-plugin.version>
-        <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
+        <maven-javadoc-plugin.version>2.10.4</maven-javadoc-plugin.version>
+        <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+        <jacoco-maven-plugin.version>0.7.9</jacoco-maven-plugin.version>
         <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
     </properties>
 
+    <repositories>
```
Comment (hameno) [inline pom.xml]: remove this, the dependencies are deployed locally

File: pom.xml [hunk 2]
```diff
@@ -70,25 +70,32 @@
...
         <java.version>1.6</java.version>
-        <android.sdk.platform>23</android.sdk.platform>
+        <android.sdk.platform>25</android.sdk.platform>
 
         <!-- Dependency Versions -->
         <junit.version>4.12</junit.version>
         <robolectric.version>3.0</robolectric.version>
-        <android.version>6.0_r3</android.version>
-        <android-support.version>23.3.0</android-support.version>
+        <android.version>4.1.1.4</android.version>
```
Comment (hameno) [inline pom.xml]: this should be 7.1.1_r3 when android.sdk.platform is 25

File: sample/res/layout/activity_sample.xml [hunk 3]
```diff
@@ -15,78 +15,97 @@
...
 
-    <Button
+    <LinearLayout 
+        android:orientation="vertical"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:text="@string/single"
-        android:onClick="onSingleClick"
-        android:layout_weight="1" />
+        android:layout_height="match_parent">
```
Comment (hameno) [inline sample/res/layout/activity_sample.xml]: the child inside a scrollview should use wrap_content for height

File: sample/src/main/java/de/psdev/licensesdialog/sample/SampleActivity.java [hunk 4]
```diff
@@ -149,4 +151,34 @@ public void onCustomThemeFragmentClick(final View view) throws Exception {
...
+        final LicensesDialogFragment fragment = new LicensesDialogFragment.Builder(this)
+            .setNotices(R.raw.notices)
+            .setNoticesCssStyle(style)
+            .build();
+
+        fragment.show(getSupportFragmentManager(), null);
+    }
+
+    private String getRGBAString(@ColorInt int color) {
+        return "background-color: rgba(" + Color.red(color) + "," + Color.green(color) + "," + Color.blue(color) + "," + ((float) Color.alpha(color) / 255) + ");";
```
Comment (hameno) [inline sample/src/main/java/de/psdev/licensesdialog/sample/SampleActivity.java]: please use String.format for this

**Comment by ChaosLeong** [comment]
@hameno Please check again.

**Comment by ChaosLeong** [comment]
Hi @hameno , will this pr be merged and release in next time?

**Comment by hameno** [comment]
Hey, yes, hopefully I have some time this weekend to do a release.

**Comment by ChaosLeong** [comment]
I hope so.

**Comment by hameno** [comment]
1.8.2 is now released

**Pull request merged**